packages/xdg_directories/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.0+3
+
+* Return null instead of throwing exception from xdg_directories/getUserDirectory when xdg-user-dir executable is missing.
```

Comment:
Nit: Remove the "xdg_directories/" from this; a package's changelog doesn't need to specify that it's about the package.

Reply:
Fixed!